sass/components/_channel-switcher.scss

```diff
@@ -50,7 +50,9 @@
 
     .channel-switcher__suggestion-box {
         position: relative;
-        height: 362px;
+        height: 100%;
```

Comment:
Is this change supposed to be part of this PR?

Reply:
same here